tests/integration/api/test_event_stream.py

```diff
@@ -86,6 +87,35 @@ def test_create_event_stream_without_credentials(
     }
 
 
+@pytest.mark.django_db
+def test_create_event_stream_with_mismatched_types(
+    admin_client: APIClient,
+    default_organization: models.Organization,
+    default_hmac_credential: models.EdaCredential,
+):
+    invalid_event_stream_type = "invalid_type"
+    data_in = {
+        "name": "test_es",
+        "event_stream_type": invalid_event_stream_type,
+        "eda_credential_id": default_hmac_credential.id,
+        "organization_id": default_organization.id,
+    }
+    with override_settings(
+        EVENT_STREAM_BASE_URL="https://www.example.com/",
+        EVENT_STREAM_MTLS_BASE_URL="https://www.example.com/",
```

Comment:
Non blocking comment - 
Wondering if this can be removed since we removed mTLS?